src/nscreg.Server.Common/Services/DataSources/UpsertUnitBulkBuffer.cs

```diff
@@ -4,8 +4,8 @@
 using System.Linq;
 using System.Threading.Tasks;
 using AutoMapper;
-using EFCore.BulkExtensions;
 using Microsoft.EntityFrameworkCore;
+using Nest;
```

Comment:
Is the `Nest` library for ElasticSearch used here?

Reply:
No. Removed.